### PR TITLE
JFG-902 Replace XML jagger configuration with java builders

### PR DESCRIPTION
**Pull request by a-badaev**

### CODE REVIEW

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestDescription.java [hunk 1]
```diff
@@ -74,6 +80,17 @@ public Builder withQueryProvider(Iterable queryProvider) {
         }
 
         /**
+         * Sets class type of {@link com.griddynamics.jagger.invoker.Invoker}.
+         * Currently available Invoker_PageVisitor.
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestDescription.java]: please, remove this line 'Currently available Invoker_PageVisitor'

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestDescription.java [hunk 2]
```diff
@@ -74,6 +80,17 @@ public Builder withQueryProvider(Iterable queryProvider) {
...
+         *
+         * @param invoker user's invoker.
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestDescription.java]: custom implementation of invoker

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestSuite.java [hunk 3]
```diff
@@ -7,22 +7,27 @@
  */
 public class JTestSuite {
 
-    private List<JTestGroup> testGroups;
 
+    private final String name;
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestSuite.java]: should be id (not name)

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestSuite.java [hunk 4]
```diff
@@ -7,22 +7,27 @@
...
+    private final List<JTestGroup> testGroups;
 
-    public static Builder builder() {
-        return new Builder();
+
+    public static Builder builder(String name) {
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestSuite.java]: it is necessary to have equal approach to set id in all builders

**Comment by anton-antonenko** [comment]
:+1:

**Pull request merged**